packages/twenty-server/src/workspace/workspace-cleaner/crons/clean-inactive-workspace.job.ts

```diff
@@ -17,10 +17,7 @@ import { DataSourceEntity } from 'src/metadata/data-source/data-source.entity';
 import { UserService } from 'src/core/user/services/user.service';
 import { EmailService } from 'src/integrations/email/email.service';
 import { EnvironmentService } from 'src/integrations/environment/environment.service';
-import {
-  FeatureFlagEntity,
-  FeatureFlagKeys,
-} from 'src/core/feature-flag/feature-flag.entity';
+import { FeatureFlagEntity } from 'src/core/feature-flag/feature-flag.entity';
```

Comment:
You can remove this line and line 49:
private readonly featureFlagRepository: Repository<FeatureFlagEntity>,